### ISSUE TITLE
Image alt text not working in post

### ISSUE THREAD

**Issue by taidang26** [elementor/hello-theme#202]
Image Alt Text alignment doesn't work. I tried to center and right align the image but the default description of the image is still on the left side.
Switching themes to other things should work properly.
![image](https://user-images.githubusercontent.com/89138954/171139296-bc4fae4c-0b93-476f-8d05-06056c1bf0d2.png)
![image](https://user-images.githubusercontent.com/89138954/171139338-6b6b80f2-4979-4abb-8a16-e8e91dfb5517.png)

**Comment by taidang26** [comment]
Hello no one here?
